shenyu-common/src/main/java/org/apache/shenyu/common/utils/UpstreamCheckUtils.java

```diff
@@ -57,32 +65,15 @@ public static boolean checkUrl(final String url) {
         }
         final boolean isHttps = url.startsWith(HTTPS);
         final int port = hostPort.length > 1 ? Integer.parseInt(hostPort[1]) : isHttps ? 443 : 80;
-        if (checkIP(hostPort[0]) || isHttps) {
```

Comment:
why remove this？

Reply:
It's useless since we remove the code `InetAddress.getByName(host).isReachable(1000)` to check url is reachable or not.

Reply:
if user config this : 192.168.0.1 ?   can check?

Reply:
We will use `80` as default port and check the port is open or not. Since the UpstreamCheckUtils is only use for divide plugin for now, I think this is reasonable. What's your opinion ?

Reply:
isHostConnector ()  can checked  ip ?  such as  192.168.0.1:8080

Reply:
Yeah, UpstreamCheckUtilsTest.testSocketConnect() contains both `ip` and `hostname` test case.